webapp/src/ts/modals/edit-message-group/edit-message-group.component.html

```diff
@@ -9,20 +9,20 @@
 >
   <form action="" method="POST">
 
-    <fieldset *ngFor="let task of model.group.rows; trackBy: trackBy" [disabled]="task.state !== 'scheduled' && task.state !== 'muted'">
+    <fieldset *ngFor="let task of model.group.rows; index as index" [disabled]="task.state !== 'scheduled' && task.state !== 'muted'">
```

Comment:
mm now that I see you removed the `trackBy` argument, I think I could do the same in other PRs I made :thinking:

Reply:
If used properly, the `trackBy` should help for performance to help Angular avoid rerendering elements that do not change. @dianabarsan is there a reason `trackBy` is undesirable here?

Reply:
There is nothing to track these elements by, unfortunately (as they come from the database). there won't be many of them at any given time, unless some configuration decides to create tens of sms-s as a result of a form and put them in the same group). 
This is an example of how one of these items looks like (taken from a random report from `gamma.dev`):
```
{
      "due": null,
      "group": 1,
      "type": "Postnatal Care from App",
      "translation_key": "schedule.pnc_from_app",
      "message_key": "messages.schedule.postnatal.day_3",
      "recipient": "clinic",
      "state_history": [
        {
          "state": "scheduled",
          "timestamp": "2020-11-20T21:26:17.742Z"
        }
      ],
    },
}
```